readme.md

```diff
@@ -428,6 +428,7 @@
 - [Text Editing](https://github.com/dok/awesome-text-editing#readme)
 - [Motion UI Design](https://github.com/fliptheweb/motion-ui-design#readme)
 - [Vue.js](https://github.com/vuejs/awesome-vue#readme) - App framework.
+- [Qwik](https://github.com/qwik-design/awesome-qwik#readme) - Resumable framework.
```

Comment:
"Resumable framework" doesn't mean anything useful without more context.

Reply:
@sindresorhus , updated

Reply:
I don't think it is updated @reemardelarosa, could you check again?

Reply:
Added it at the end of Frontend
<img width="1473" alt="Screenshot 2022-11-09 at 8 53 07 PM" src="https://user-images.githubusercontent.com/4918140/200835668-cdd9372e-28eb-4e3a-97d8-d4f974c15231.png">

Reply:
Hi there, adding the word "web" doesn't make it anymore descriptive. Most users wouldn't know what "Resumable" refers to.